.gitignore

```diff
@@ -1,4 +1,4 @@
-charts/*/charts
+charts/**/*.tgz
```

Comment:
unclear if we need to check in these .tgz archives. i believe it's just an optimization to pull down chart.

https://helm.sh/docs/topics/charts/#managing-dependencies-with-the-dependencies-field